.gitlab-ci.yml

```diff
@@ -339,6 +340,13 @@ init_workspace:
       git checkout -f -b pr ${MTLS_AMBASSADOR_REV})
     - (cd go/src/github.com/mendersoftware/mtls-ambassador && echo -e "# $(git log -n1 --oneline)\nexport MTLS_AMBASSADOR_REV=$MTLS_AMBASSADOR_REV\nexport MTLS_AMBASSADOR_REV_GIT_SHA=$(git rev-parse HEAD)" >> ${CI_PROJECT_DIR}/build_revisions.env)
 
+    - git clone git@github.com:mendersoftware/deviceconnect go/src/github.com/mendersoftware/deviceconnect
```

Comment:
Not very important, but for the open source repos we have been using `https` for the clone instead of `git`.

Please change it to http for consistency. Alternatively, maybe is time to align all to git?